mrtt-ui/src/views/LandscapeForm.js

```diff
@@ -135,6 +153,22 @@ const LandscapeForm = ({ isNewLandscape }) => {
   const handleCancelClick = () => {
     navigate('/landscapes')
   }
+
+  const handleDeleteClick = () => {
+    setIsDeleting(true)
+    axios
+      .delete(landscapeUrl)
+      .then(() => {
+        setIsDeleting(false)
+        toast.success(language.success.getDeleteThingSuccessMessage(landscapeName))
```

Comment:
what thing are we referring to ? can it be more specific?

Reply:
Im just reusing the same function to return strings

```
 getEditThingSuccessMessage: (thing) => `${thing} has been edited.`,
  getCreateThingSuccessMessage: (thing) => `${thing}, has been created.`,
  getDeleteThingSuccessMessage: (thing) => `${thing} has been deleted.`
  ```

Reply:
Rather than just having a 500 functions. For timelines.

Reply:
ah ok got it!